web/tsconfig.json

```diff
@@ -11,9 +11,8 @@
     "outDir": "dist",
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
+    "types": ["cypress", "node"]
   },
-  "include": [
-    "src"
-  ],
-  "exclude": [ "node_modules" ]
+  "include": ["src", "**/*.ts"],
```

Comment:
shouldn't we have all *.ts files in src/?

Reply:
yeah, but for some reason my linter was showing me errors for *.cy.ts files without this include

Reply:
ah, ok, cypress tests are in cypress/e2e/, not src/